setup.py

```diff
@@ -99,7 +99,14 @@ def collectExtraFiles():
             "sphinx-gallery",
         ],
     },
-    tests_require=["nbconvert", "jupyter_client", "ipykernel"],
+    tests_require=[
+        "nbconvert",
+        "jupyter_client",
+        "ipykernel",
+        "wxpython",
+        "pytest-xvfb",
+        "screeninfo",
+    ],
```

Comment:
I'm not sure if we still need `tests_require` at all, to be honest. According to the `setuptools` docs:

```
tests_require

    If your project’s tests need one or more additional packages besides those needed to install it,
you can use this option to specify them. It should be a string or list of strings specifying what
other distributions need to be present for the package’s tests to run. When you run the test
command, setuptools will attempt to obtain these (using pip if available). Note that these
required projects will not be installed on the system where the tests are run, but only
downloaded to the project’s setup directory if they’re not already installed locally.

    New in 41.5.0: Deprecated the test command.

```

Looks like it's only used by the `setup.py test` command, which has been deprecated anyways. It's unclear to me if anything else looks at this (e.g., pip)... I think the `requirements-testing.txt` might be sufficient to specify extra dependencies, without the duplication, but it does seem weird to not express them in the `setup.py`... Thoughts?